Frontend/android-native/app/src/main/java/in/juspay/mobility/utils/LocationUpdateService.java

```diff
@@ -368,8 +371,154 @@ private void updateDriverStatus(Boolean status) {
         });
     }
 
+    private JSONObject getLatLng(Double lat , Double lng) throws JSONException {
+        JSONObject latLng = new JSONObject();
+        latLng.put("lat", lat);
+        latLng.put("lon", lng);
+        return latLng;
+    }
+
+    private void getRoute(Double startLat, Double startLng, Double endLat, Double endLng) {
+        StringBuilder result = new StringBuilder();
+
+        SharedPreferences sharedPref = context.getSharedPreferences(context.getString(R.string.preference_file_key), Context.MODE_PRIVATE);
+
+        String regToken = sharedPref.getString("REGISTERATION_TOKEN", "null");
+        String baseUrl = sharedPref.getString("BASE_URL", "null");
+        String version = sharedPref.getString("VERSION_NAME", "null");
+        try {
+            String url = baseUrl + "/trip/route";
+            HttpURLConnection connection = (HttpURLConnection) (new URL(url).openConnection());
+            connection.setRequestMethod("POST");
```

Comment:
Change requestMethod to `GET` in purescript also switch the `/route` api methods to `GET` from `POST`